release notes/v0.35.0.md

```diff
@@ -0,0 +1,97 @@
+k6 v0.35.0 is here! :tada: It introduces several new features that nicely enhance usage of k6 as well as contains a whole number of fixes and ongoing efforts with refactoring.
+
+In total, we have closed 14 issues. We have also branched out three new xk6 extensions during this release :star:
+
+## New features
+
+### VU tags ([#2172](https://github.com/grafana/k6/pull/2172))
+
+k6 now supports setting tags for VUs as part of the Execution API with an easy key-value interface. These tags are attached to the metrics emitted by the VU. Example usage:
+
+```js
+import exec from 'k6/execution';
+import { sleep } from 'k6';
+
+export const options = {
+    duration: '10s',
+    vus: 3,
+};
+
+export default function () {
+    exec.vu.tags['mytag'] = 'value';
+    exec.vu.tags['vuId'] = exec.vu.idInTest;
+    console.log(`mytag is ${exec.vu.tags['mytag']} and my VU's ID in tags ${exec.vu.tags['vuId']}`);
+    sleep(5);
+}
+```
+
+There are some caveats, however: values can be only of `String`, `Number` or `Boolean` type, while values of other types will result either in a warning or an exception. Additionally, given that k6 has a whole bunch of system tags, one should be careful with using them as keys. You can read complete information about VU tags in [`k6/execution docs`](https://k6.io/docs/javascript-api/k6-execution/#vu).
+
+### JS promises are now an option 
+
+With the goja update in [#2197](https://github.com/grafana/k6/pull/2197), you can now make a `Promise` and chaining in your k6 scripts:
+
+```js
+export default function () {
+    var p = new Promise((resolve, reject) => {
+        console.log('do something promising!');
+        reject('here');
+    });
+
+    p.then(
+        (s) => { console.log('fulfilled with', s) },
+        (s) => { console.log('rejected with', s) },
+    );
+}
+```
+
+It must be noted that `Promise`s are not used by k6 itself yet but it is a stepping stone for implementing async functionality in the future releases.
+
+### InfluxDB v2 API
+
+After hard work at working out how to integrate InfluxDB v2 API, it was decided to pull that integration into an [xk6-output-influxdb extension](https://github.com/grafana/xk6-output-influxdb). InfluxDB v1 API is still available as before, with minor optimizations ([#2190](https://github.com/grafana/k6/pull/2190)).
+
+### New xk6 extensions
```

Comment:
:thinking: this should probably be its own major section, and it should probably be below `Other notable changes`. Also, the actual extension names should be more prominent, headers in their own right instead of a simple list. And `xk6-output-influxdb` should probably be here, not in the changes above.

Something like this:
```
## New features
### Ability to set VU-wide custom metric tags
### Initial basic support for JS promises
### Support gRPC server reflection
### Other notable changes and improvements

## New xk6 extensions
### xk6-browser
### xk6-output-remote-write
### xk6-output-influxdb
```